superset-frontend/src/cccs-viz/plugins/plugin-chart-at-a-glance-user-id/src/AtAGlanceUserID.tsx

```diff
@@ -254,13 +246,7 @@ function AtAGlanceUserIDCore(initialFormData: QueryFormData) {
                 {aadDataManager.isLoading && !aadDataManager.isInit ? (
                   <></>
                 ) : (
-                  <ul>
-                    {unsuccessfulNonCanadianIpsList.map(
-                      (a: { client_ip: string }) => (
-                        <li>{a.client_ip}</li>
-                      ),
-                    )}
-                  </ul>
+                  generateClientIpLinksList(unsuccessfulNonCanadianIpsList, props.ipDashBoardBaseUrl, props.ipDashboardId ,props.ipDashboardFilterId)
```

Comment:
I'm ok with this but you probably could've gotten away with just passing a reference to props as the second argument. :)
That being said the last argument...  could you move the comma back one space and then put a space to the right of it?

Reply:
Linter must have missed it! and yea I passed those in to like decouple it from the props / component if we want to reuse but that does seem unlikely